src/aerialmap_display.cpp

```diff
@@ -131,65 +133,168 @@ void AerialMapDisplay::subscribe()
 void AerialMapDisplay::unsubscribe()
 {
   coord_sub_.shutdown();
-  ROS_INFO("Unsubscribing.");
 }
 
 void AerialMapDisplay::updateAlpha()
 {
-  alpha_ = alpha_property_->getFloat();
+  // if draw_under_ texture property changed, we need to
```

Comment:
not sure if I should keep those, but it helped a lot to figure out what needs to be called when

Reply:
Please keep it, it definitely will serve us in the future, when we forgot about the code logic again :)